easybuild/easyconfigs/s/Spyder/Spyder-4.1.5-foss-2019a-Python-3.7.2.eb

```
@@ -0,0 +1,327 @@
+        'modulename': 'xdg',
+        'checksums': ['fe2928d3f532ed32b39c32a482b54136fe766d19936afc96c8f00645f9da1a06'],
+    }),
+    ('requests', '2.23.0', {
+        'checksums': ['b3f43d496c6daba4493e7c431722aeb7dbc6288f52a6e04e7b6023b0247817e6'],
+    }),
+    ('rope', '0.17.0', {
+        'checksums': ['658ad6705f43dcf3d6df379da9486529cf30e02d9ea14c5682aa80eb33b649e1'],
+    }),
+    ('QDarkStyle', '2.8.1', {
+        'checksums': ['d53b0120bddd9e3efba9801731e22ef86ed798bb5fc6a802f5f7bb32dedf0321'],
+    }),
+    ('QtAwesome', '0.7.2', {
+        'checksums': ['ca9719c82d41707f62c340811b23bcab95336e73edd88b7eab7fd951d2e27fab'],
+    }),
+    ('qtconsole', '4.7.7', {
+        'checksums': ['f236ead8711dba0702507dd8fad473c7216a86eefa6098eff8ec4b54f57d7804'],
+    }),
+    ('QtPy', '1.9.0', {
+        'checksums': ['2db72c44b55d0fe1407be8fba35c838ad0d6d3bb81f23007886dc1fc0f459c8d'],
+    }),
+    ('SecretStorage', '3.1.2', {
+        'modulename': 'secretstorage',
+        'checksums': ['15da8a989b65498e29be338b3b279965f1b8f09b9668bd8010da183024c8bff6'],
+    }),
+    ('snowballstemmer', '2.0.0', {
+        'checksums': ['df3bac3df4c2c01363f3dd2cfa78cce2840a79b9f1c2d2de9ce8d31683992f52'],
+    }),
+    ('sortedcontainers', '2.2.2', {
+        'checksums': ['4e73a757831fc3ca4de2859c422564239a31d8213d09a2a666e375807034d2ba'],
+    }),
+    ('Sphinx', '3.2.1', {
+        'checksums': ['321d6d9b16fa381a5306e5a0b76cd48ffbc588e6340059a729c6fdd66087e0e8'],
+    }),
+    ('sphinxcontrib-applehelp', '1.0.2', {
+        'modulename': 'sphinxcontrib.applehelp',
+        'checksums': ['a072735ec80e7675e3f432fcae8610ecf509c5f1869d17e2eecff44389cdbc58'],
+    }),
+    ('sphinxcontrib-devhelp', '1.0.2', {
+        'modulename': 'sphinxcontrib.devhelp',
+        'checksums': ['ff7f1afa7b9642e7060379360a67e9c41e8f3121f2ce9164266f61b9f4b338e4'],
+    }),
+    ('sphinxcontrib-htmlhelp', '1.0.3', {
+        'modulename': 'sphinxcontrib.htmlhelp',
+        'checksums': ['e8f5bb7e31b2dbb25b9cc435c8ab7a79787ebf7f906155729338f3156d93659b'],
+    }),
+    ('sphinxcontrib-jsmath', '1.0.1', {
+        'modulename': 'sphinxcontrib.jsmath',
+        'checksums': ['a9925e4a4587247ed2191a22df5f6970656cb8ca2bd6284309578f2153e0c4b8'],
+    }),
+    ('sphinxcontrib-qthelp', '1.0.3', {
+        'modulename': 'sphinxcontrib.qthelp',
+        'checksums': ['4c33767ee058b70dba89a6fc5c1892c0d57a54be67ddd3e7875a18d14cba5a72'],
+    }),
+    ('sphinxcontrib-serializinghtml', '1.1.4', {
+        'modulename': 'sphinxcontrib.serializinghtml',
+        'checksums': ['eaa0eccc86e982a9b939b2b82d12cc5d013385ba5eadcc7e4fed23f4405f77bc'],
+    }),
+    ('spyder', version, {
+        'preinstallopts': """sed -i "s/'pyqt5<5.13;python_version>=\"3\"',//g" setup.py && sed -i "s/'pyqtwebengine<5.13;python_version>=\"3\"',//g" setup.py && sed -i 's/jedi==0.17.1/jedi==0.17.2/g' setup.py && sed -i "s/JEDI_REQVER = '=0.17.1'/JEDI_REQVER = '=0.17.2'/g" spyder/dependencies.py && """,
```

@zarybnicky Lines must fit in 120-char columns or the style check fails, so how about something like this (untested):

```suggestion
        'preinstallopts': ' && '.join([
            """sed -i "s/'pyqt5<5.13;python_version>=\"3\"',//g" setup.py""",
            """sed -i "s/'pyqtwebengine<5.13;python_version>=\"3\"',//g" setup.py""",
            """sed -i 's/jedi==0.17.1/jedi==0.17.2/g' setup.py""",
            """sed -i "s/JEDI_REQVER = '=0.17.1'/JEDI_REQVER = '=0.17.2'/g" spyder/dependencies.py""",
        ]),
```